<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COwnModelSetForm</class>
 <widget class="QFrame" name="COwnModelSetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="vl_OwnModelSetForm">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_Selection">
     <property name="title">
      <string>Selection</string>
     </property>
     <layout class="QGridLayout" name="gl_Selection" columnstretch="0,1,1,1">
      <item row="0" column="1" colspan="3" alignment="Qt::AlignLeft">
       <widget class="swift::gui::components::CSimulatorSelector" name="comp_SimulatorSelector">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_Simulator">
        <property name="text">
         <string>Simulator:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="rb_Full">
        <property name="toolTip">
         <string>new set, existing data will be deleted</string>
        </property>
        <property name="text">
         <string>full</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bg_Mode</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_Mode">
        <property name="text">
         <string>Mode:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_SelectSourceSet">
        <property name="text">
         <string>Source set:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="rb_DbIcaoCodesOnly">
        <property name="toolTip">
         <string>Model has ICAO code known in DB</string>
        </property>
        <property name="text">
         <string>DB  ICAO data</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bg_SourceSet</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="rb_Incremental">
        <property name="toolTip">
         <string>add to existing set</string>
        </property>
        <property name="text">
         <string>incremental</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bg_Mode</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QRadioButton" name="rb_WithIcaoData">
        <property name="toolTip">
         <string>Model has ICAO code, but not necessarily valid data</string>
        </property>
        <property name="text">
         <string> ICAO data</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bg_SourceSet</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="cb_ConsolidateModelSet">
        <property name="text">
         <string>consolidate with DB</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="rb_DbDataOnly">
        <property name="toolTip">
         <string>Own models for which DB data are available</string>
        </property>
        <property name="text">
         <string>DB data only</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bg_SourceSet</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QRadioButton" name="rb_DistributorsFromBelow">
        <property name="toolTip">
         <string>all distributors shown below</string>
        </property>
        <property name="text">
         <string>all shown below</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bg_Distributors</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_Providers">
        <property name="text">
         <string>Distributors:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="rb_DistributorsSelected">
        <property name="toolTip">
         <string>all selected distributors from below</string>
        </property>
        <property name="text">
         <string>selected from below</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bg_Distributors</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QRadioButton" name="rb_DistributorsAll">
        <property name="text">
         <string>all distributors</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bg_Distributors</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_Distributors">
     <property name="title">
      <string>Distributor selection</string>
     </property>
     <layout class="QVBoxLayout" name="vl_Distributors">
      <item>
       <widget class="QWidget" name="wi_DistributionButtons" native="true">
        <layout class="QGridLayout" name="gl_DistributorButtons">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item row="0" column="0">
          <widget class="QRadioButton" name="rb_DisplayPreferencesDistributors">
           <property name="text">
            <string>from preferences</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_Displayed</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="rb_DisplayAllDistributors">
           <property name="text">
            <string>all distributors</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_Displayed</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="cb_SortByPreferences">
           <property name="toolTip">
            <string>If checked the distributors from the settings page will be used</string>
           </property>
           <property name="text">
            <string>sort by dist. preferences</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="swift::gui::views::CDistributorView" name="tvp_Distributors">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::components::CSimulatorSelector</class>
   <extends>QFrame</extends>
   <header>gui/components/simulatorselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::views::CDistributorView</class>
   <extends>QTableView</extends>
   <header>gui/views/distributorview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rb_Full</tabstop>
  <tabstop>rb_Incremental</tabstop>
  <tabstop>cb_ConsolidateModelSet</tabstop>
  <tabstop>rb_DbDataOnly</tabstop>
  <tabstop>rb_DbIcaoCodesOnly</tabstop>
  <tabstop>rb_WithIcaoData</tabstop>
  <tabstop>rb_DistributorsSelected</tabstop>
  <tabstop>rb_DistributorsFromBelow</tabstop>
  <tabstop>rb_DistributorsAll</tabstop>
  <tabstop>rb_DisplayPreferencesDistributors</tabstop>
  <tabstop>rb_DisplayAllDistributors</tabstop>
  <tabstop>cb_SortByPreferences</tabstop>
  <tabstop>tvp_Distributors</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bg_Mode"/>
  <buttongroup name="bg_Displayed"/>
  <buttongroup name="bg_SourceSet"/>
  <buttongroup name="bg_Distributors"/>
 </buttongroups>
</ui>
